km_api/know_me/tests/models/test_km_user_model.py

```diff
@@ -1,31 +1,107 @@
+from rest_framework.reverse import reverse
+
 from know_me import models
 
 
 def test_create(user_factory):
     """
-    Test creating a KMUser.
+    Test creating a km_user.
     """
     models.KMUser.objects.create(
-       image=None,
-       quote='kuh mooz er',
-       user=user_factory())
+        quote='Life is like a box of chocolates.',
+        user=user_factory())
 
 
-def test_name(km_user_factory):
```

Comment:
Cool, my one concern is that test_name and test_string_conversion will be essentially identical, is that ok?